cpp/log/cert.cc

```diff
@@ -883,7 +914,8 @@ Cert::Status Cert::IsValidNameConstrainedIntermediateCa() const {
 
   // If it's not a CA cert or there is no name constraint extension then we
   // don't need to apply the rules any further
-  if (HasBasicConstraintCATrue() != Cert::TRUE
+  StatusOr<bool> has_ca_constraint_status = HasBasicConstraintCATrue();
+  if (StatusOrBoolToCertStatus(has_ca_constraint_status) != Cert::TRUE
```

Comment:
It seems to map directly enough, from "not `Cert::TRUE` to "ok and not `true`" (the checking for `ok()` is really because of the `OrDie` part of `ValueOrDie`, but if you ignore that, it's kind of exactly what it used to say).